### PR TITLE
ENH: plot_roi with contours

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Discussed IRL with @GaelVaroquaux

### CODE REVIEW

**Comment by KamalakerDadi** [comment]
Still-to-do. More tests:

- testing _plot_roi_contours
- testing whether raises an error with wrong view_type

File: nilearn/plotting/img_plotting.py
Comment (KamalakerDadi) [inline nilearn/plotting/img_plotting.py]: What would be the good default ```levels``` threshold for labels based ROIs ?

**Comment by GaelVaroquaux** [comment]
Can you rebase / merge on master, so that circle runs.

File: nilearn/plotting/img_plotting.py
Comment (GaelVaroquaux) [reply nilearn/plotting/img_plotting.py]: I think that the one that you used (.5) is good.

File: nilearn/plotting/tests/test_img_plotting.py
Comment (GaelVaroquaux) [inline nilearn/plotting/tests/test_img_plotting.py]: I think that I would prefer it if you used the public-facing function in the test.

**Comment by GaelVaroquaux** [comment]
Also: this is cool! Can you add it to the whats_new.

**Comment by GaelVaroquaux** [comment]
Can you merge with master, so that CI runs without failing. Thanks!

File: nilearn/plotting/tests/test_img_plotting.py
Comment (jeromedockes) [reply nilearn/plotting/tests/test_img_plotting.py]: @KamalakerDadi I think this is the last comment to address before this is merged?

> I think that I would prefer it if you used the public-facing function in the test.

**Comment by thomasbazeille** [comment]
hello @KamalakerDadi ! Everything seemed alright for this PR, is there any particular reason why it was not merged ?

**Comment by thomasbazeille** [comment]
hello @KamalakerDadi ! Apart for a few conflicts, everything seemed alright for this PR. Can it be finished and merged to be included in the upcoming release ?

**Comment by KamalakerDadi** [comment]
>hello @KamalakerDadi ! Apart for a few conflicts, everything seemed alright for this PR. Can it be finished and merged to be included in the upcoming release ?

It's doable. Let me get back to this very soon.

**Comment by KamalakerDadi** [comment]
@thomasbazeille please let me know if anything needs to be done.

**Comment by thomasbazeille** [comment]
I didn't see it added in the what's new; apart from that everything seems there after a quick review. Will merge in a few days if I don't here from anyone.

Thanks a lot for completing this great PR Kamalakar!

**Review by thomasbazeille — APPROVED** [review]

**Review by bthirion — APPROVED** [review]
Indeed a whatsnew entry would be ideal.

**Pull request merged**
